### PR TITLE
Update the WSL guideline

### PR DESCRIPTION
**Pull request by dend**
MongoDB in WSL is supported - linked to the [official documentation](https://docs.microsoft.com/windows/wsl/tutorials/wsl-database#install-mongodb) as well. I just updated the admonition with a link to the document above.

### CODE REVIEW

**Comment by dend** [comment]
@andf-mongodb - can you please review and see if I need to tweak anything to merge this in? Want to make sure that folks are able to use the product in WSL.

**Comment by andf-mongodb** [comment]
Thanks @dend. I can't direct users to your linked resource until it is updated to MDB v4.4 (or v5.0 in two weeks). Currently your instructions install v3.6. Background: {{mongodb}} package is hosted by Ubuntu and is wildly out of date. {{mongodb-org}} (MDB Community) or {{mongodb-enterprise}} (MDB Enterprise) are the MDB-hosted and latest version metapackages. Are you able to update your doc? Do you need a hand / peer reviewer? Thank you.

**Comment by dend** [comment]
Thanks @andf-mongodb - will work on this with @mattwojo. Will keep this PR open and keep you posted on our progress.

**Comment by mattwojo** [comment]
@andf-mongodb / @dend - I updated our docs for WSL to reflect the install steps for MDB v5.0: https://docs.microsoft.com/windows/wsl/tutorials/wsl-database#install-mongodb

Works great on my machine, but please take a look and let me know if you have any questions or concerns.
Thanks!!

**Comment by dend** [comment]
Thanks for the ping @mattwojo - I followed the instructions and got MongoDB running in WSL.

@andf-mongodb - let us know if the changes look good, and whether we need to add any other details to make this PR ready for merging.

**Comment by andf-mongodb** [comment]
Thank you -- the linked instructions now look good to me, thank you for the update!

I am leaving MDB shortly, so let me hand off this PR to the team to handle the change. I will recommend at least a removal of the "do not use" messaging, but am not sure of company stance on linking out to third party instructions.

Thanks again for your work on this!

**Comment by dend** [comment]
Sounds good @andf-mongodb. If it helps, I am happy to create an explicit PR here with instructions for WSL.

**Comment by dend** [comment]
Tagging @ianf-mongodb and @jeff-allen-mongo to see if there is anything we can do to help move this forward.

File: source/includes/admonition-wsl.rst [hunk 1]
```diff
@@ -1,4 +1,4 @@
-.. warning:: Windows Subsystem for Linux (WSL) - Unsupported
+.. note:: Windows Subsystem for Linux (WSL) Support
 
 
-   MongoDB does not support the Windows Subsystem for Linux (WSL).
\ No newline at end of file
+   To run MongoDB in Windows Subsystem for Linux (WSL), refer to the `WSL documentation <https://docs.microsoft.com/windows/wsl/tutorials/wsl-database#install-mongodb>`.
```
Comment (jeff-allen-mongo) [inline source/includes/admonition-wsl.rst:4]: Just two things to change:

- Line break at 72 chars
- We need a double underscore after a link that goes outside of the docs site.

See the following suggestion:

```suggestion
   To run MongoDB in Windows Subsystem for Linux (WSL), refer to the
   `WSL documentation <https://docs.microsoft.com/windows/wsl/tutorials/wsl-database#install-mongodb>`__.
```

**Review by jeff-allen-mongo — COMMENTED** [review]
Thanks for the ping, @dend. I left a comment regarding formatting, and once that is addressed I can merge this in. Thank you.

**Comment by dend** [comment]
@jeff-allen-mongo - done, thank you so much for the suggestion!

**Comment by jeff-allen-mongo** [comment]
Thanks for updating @dend. I've merged this change in. Cheers!

**Pull request merged**